package multiple.catch_block.progrm.flow;

public class Test {

	public static void main(String[] args) throws ArithmeticException {
		
		try {
			System.out.println(10/0);
		}
		catch(ArithmeticException e) {
			
		}
		catch(Exception e) {
			
		}
		

	}

}
